stream_alert/shared/config.py

```diff
@@ -33,6 +33,24 @@ class TopLevelConfigKeys(object):
 class ConfigError(Exception):
     """Exception class for config file errors"""
 
+class SchemaSorter(object):
+    """Sort Schema by Priority"""
+
+    # Set a default index to -1
+    max_index = -1
+
+    @classmethod
+    def sort(cls, key_and_value_tuple):
+        dict_value = key_and_value_tuple[1]
+        value = int(dict_value.get('configuration', {}).get('priority', -1))
+
+        # Update the index to the max of the current index or cached one
+        cls.max_index = max(cls.max_index, value)
```

Comment:
Why do we need static statefulness in a comparison function? This seems super dangerous as it'd make successive iterations of this function to be buggy